### DISCUSSION TITLE
Mod Difficulty Adjust Save Presets

### DISCUSSION THREAD

**Discussion by tylort69** [ppy/osu#16276] [Ideas] [answered]
![image](https://user-images.githubusercontent.com/28652424/147652159-00f3e241-1aa4-4b2f-aef0-f24d1644f8fd.png)
I think that it would be a very nice QOL feature to allow users to save presets for the difficulty adjust mod allowing to switch between different configurations to practice more skills easier. If that's too much work than maybe at least saving the last settings that you had it set to be persistent through restarts.

**Comment by peppy** [comment, marked as answer]
Mod presets will be coming with an updated mod select interface
